build/common.sh

```diff
@@ -124,6 +125,7 @@ kube::build::get_docker_wrapped_binaries() {
     "kube-controller-manager,${KUBE_CONTROLLER_MANAGER_BASE_IMAGE}"
     "kube-scheduler,${KUBE_SCHEDULER_BASE_IMAGE}"
     "kube-proxy,${KUBE_PROXY_BASE_IMAGE}"
+    "kubectl,${KUBELET_BASE_IMAGE}"
```

Comment:
Curious why this is defined in terms of kubelet?

Reply:
+1, this looks like a typo.